### ISSUE TITLE
Some issues with c2m6_a

### ISSUE THREAD

**Issue by x3003** [Realm667/WolfenDoom#1364]
This tree is half on the path

![Screenshot_Doom_20240101_172301](https://github.com/Realm667/WolfenDoom/assets/77861029/e313ce73-5768-43fc-a29f-fbb357cc0d6e)

If you go forward w/ this type of table (on the long side), you can't walk over it, you have to jump. But when walking from the (short) side, there's no problem. Maybe there's a height problem ?

![Screenshot_Doom_20240101_173206](https://github.com/Realm667/WolfenDoom/assets/77861029/32cbafc3-8d15-4190-835e-02b6872e6044)

This texture (2x) at the back of the building is wrong.

![Screenshot_Doom_20240101_174003](https://github.com/Realm667/WolfenDoom/assets/77861029/b75108c7-bd49-4f8e-a642-6e6e5aafb3ca)

**Comment by hawkwind3** [comment]
That 3rd one with the sky on the slope. I had reported that issue earlier but back then it was rendering a spurious vertical grey texture. It is a mapping error which can be fixed.

**Comment by hawkwind3** [comment]
Tree issue ...

Much better but 2 tree models at sector 1550 partly render into sector 1544. More importantly, look up at sector 1544 and can partly see the top of trees when viewing either sectors 1551 or 1552. Skywall issue.

Table issue ...

The lines I mentined before are fixed but also line 11703 needs to be flagged sector type 3 - non-solid, as have to jump at line 11711.

Slope issue ...

Confirmed as fixed.
